src/software/ai/hl/stp/tactic/goalie/simulated_goalie_tactic_test.cpp

```diff
@@ -0,0 +1,327 @@
+#include <gtest/gtest.h>
+
+#include <utility>
+
+#include "software/ai/hl/stp/tactic/goalie/goalie_tactic.h"
+#include "software/ai/hl/stp/tactic/move/move_tactic.h"
+#include "software/geom/algorithms/contains.h"
+#include "software/simulated_tests/non_terminating_validation_functions/enemy_never_scores_validation.h"
+#include "software/simulated_tests/simulated_tactic_test_fixture.h"
+#include "software/simulated_tests/terminating_validation_functions/ball_kicked_validation.h"
+#include "software/simulated_tests/terminating_validation_functions/robot_received_ball_validation.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/world/world.h"
+
+class SimulatedGoalieTacticTest : public SimulatedTacticTestFixture
+{
+   protected:
+    void checkGoalieSuccess(int seconds_to_wait, std::shared_ptr<World> world_ptr,
+                            ValidationCoroutine::push_type& yield)
+    {
+        auto initial_time = world_ptr->getMostRecentTimestamp();
+        while (world_ptr->getMostRecentTimestamp() <
+               initial_time + Duration::fromSeconds(seconds_to_wait))
+        {
+            yield("Waiting " + std::to_string(seconds_to_wait) + " seconds to check that the enemy team did not score");
+        }
+    }
+    void SetUp() override
+    {
+        SimulatedTacticTestFixture::SetUp();
+        addEnemyRobots(TestUtil::createStationaryRobotStatesWithId(
+            {Point(1, 0), Point(1, 2.5), Point(1, -2.5), field().enemyGoalCenter(),
+             field().enemyDefenseArea().negXNegYCorner(),
+             field().enemyDefenseArea().negXPosYCorner()}));
+    }
+};
+
+TEST_F(SimulatedGoalieTacticTest, DISABLED_test_panic_ball_very_fast_in_straight_line)
```

Comment:
I'm also curious about why some of these tests are disabled

Reply:
Good catch, that was just so when I ran the tests I could skip a bunch of them, I just forgot to un-disable them afterwards!